web/files/assets/css/custom.css

```diff
@@ -120,38 +102,48 @@ table tr th :last-child, table tr td :last-child {
     }
 }
 
+/* Graphics of the "feature card" to be placed in the .feature-column
+ *   Intended to be used with <span> so it can be nested inside a
+ *   hyperlink <a>
+ *
+ *   By default the PlasmaPy graphic will be placed as a background
+ *   image, but this can be changed by specifying a new value for the
+ *   background-image property.
+ */
```

Comment:
As someone to whom css is magic, thank you, o great wizard!

Reply:
Yeah, it's always been magic to me too.  This site helped me so much...

https://www.w3schools.com/howto/default.asp